perceval/backends/core/meetup.py

```diff
@@ -92,17 +91,10 @@ def __init__(self, group, api_token, is_oauth_token=False,
         self.group = group
         self.max_items = max_items
         self.api_token = api_token
-        self.is_oauth_token = is_oauth_token
         self.sleep_for_rate = sleep_for_rate
         self.min_rate_to_sleep = min_rate_to_sleep
         self.sleep_time = sleep_time
 
-        if not self.is_oauth_token:
-            warnings.warn(
```

Comment:
There's also some references to `warning` on the top of the file that you should remove.